prog/dftb+/lib_dftb/nonscc.F90

```diff
@@ -303,6 +322,9 @@ subroutine buildDiatomicBlocks(out, skCont, coords, nNeighbors, &
     end do
     !$OMP  END PARALLEL DO
 
+    ! Note
+    ! Do not assemble chunks back together, as its done in routine calling this one
+
```

Comment:
Routine should rather receive the boundaries as dummy argument. That is more explicit and causes less confusion. (I've changed it accordingly).